src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/EventDTO.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import java.time.OffsetDateTime
+
+data class EventDTO(
+  val eventType: String,
+  val description: String,
+  val detailUrl: String,
+  val occurredAt: OffsetDateTime,
+  val version: Int,
+  val additionalInformation: Map<String, Any>
+)
```

Comment:
Should additional be defaulted to an empty list?
Any looks a bit strange but can't think of anything better